eventually/vote/models.py

```diff
@@ -189,9 +223,21 @@ class Answer(models.Model):
     """
     vote = models.ForeignKey(Vote)
```

Comment:
if u delete vote, answer should be deleted too -> on_delete=models.CASCADE

Reply:
Can Null be True here?